crawl-ref/source/acquire.cc

```diff
@@ -549,7 +549,7 @@ static int _acquirement_missile_subtype(bool /*divine*/, int & /*quantity*/)
 
     switch (skill)
     {
-    case SK_SLINGS:    result = MI_SLING_BULLET; break;
+	case SK_SLINGS:    result = MI_STONE; break;
     case SK_BOWS:      result = MI_ARROW; break;
     case SK_CROSSBOWS: result = MI_BOLT; break;
```

Comment:
The whitespace that needs fixing is here: please change the tab to four spaces.

Reply:
Uh... God, I was looking for the wrong place at all the wrong!